services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java

```diff
@@ -2314,11 +2315,18 @@ private Node getRootFolderNode(String identity, String type) throws RepositoryEx
     // TODO use the user session, not a system session
     SessionProvider sessionProvider = sessionProviders.getSystemSessionProvider(null);
     Session session = sessionProvider.getSession(repository.getConfiguration().getDefaultWorkspaceName(), repository);
-    if (OWNER_TYPE_SPACE.equals(type) || OWNER_TYPE_SPACEEVENT.equals(type)) {
+    if (OWNER_TYPE_SPACE.equals(type)) {
       Node rootSpace = null;
       Space space = spaceService.getSpaceByPrettyName(identity);
       rootSpace = (Node) session.getItem(nodeCreator.getJcrPath(CMS_GROUPS_PATH) + space.getGroupId());
       folderNode = rootSpace.getNode("Documents");
+    } else if (OWNER_TYPE_SPACEEVENT.equals(type)) {
+      Node rootSpace = null;
+      IdentityManager identityManager = CommonsUtils.getService(IdentityManager.class);
```

Comment:
Service already declared socialIdentityManager

Reply:
Ok, i changed it